src/pages/getstarted/LearningList.scss

```diff
@@ -0,0 +1,6 @@
+.learning-items {
```

Comment:
Please use inline css, no external css file

Reply:
Move all png files inside the png folder not in the svg folder

Reply:
Thanks Ed,
-External css file: as far as i could see I couldn't implement a 'box shadow on hover' using inline css, but I will look into this again to make sure.
-.png files: i used png for now as SVG are not supported by the gatsby-image plugin (i guess as they are already responsive). I will look into this further to find another way so that i can use the svg images...

Reply:
Here are some code to do the hover state:

````
<div
  css={{
    ':hover': {
      box-shadow: 1px 1px 10px 1px gray-400;
    }
  }}
>
</div>
````

About the png, don't use png at all.

Reply:
Thanks Ed! Working on it...

Reply:
@eouin Just committed those changes and they are ready for review. Thanks